vitrina/settings.py

```diff
@@ -246,6 +246,7 @@
 EMAIL_HOST_USER = ''
 EMAIL_HOST_PASSWORD = ''
 EMAIL_PORT = '2525'
+DEFAULT_FROM_EMAIL = 'Test Domain <noreply@testdomain.com>'
```

Comment:
```suggestion
DEFAULT_FROM_EMAIL = 'Test Domain <noreply@example.com>'
```

example.com is specifically reserved for testing purposes.